configuration/observer.go

```diff
@@ -73,9 +73,9 @@ func (Observer) Default() *Observer {
 			CacheSize:           10000,
 			Auth: Auth{
 				Required:      true,
-				URL:           "https://{access_token_url}",
-				Login:         "{unique_login}",
-				Password:      "{strong_password}",
```

Comment:
${unique_login} I think it's better to keep placeholders and not default values

Reply:
Is "<your_login>" a correct placeholder in this case?
Or does the language enforce this type of syntax ${your_login}?

Reply:
Made them empty according to the resolution found in our Slack conversation.